web.main.js

```diff
@@ -14,15 +14,21 @@ var minify = require('gulp-minify');
 var cleanCss = require('gulp-clean-css');
 const path = require('path');
 
+var i18n = {
```

Comment:
I believe this is already specified in the ```build.js``` file. Maybe we should move this information to ```main.js``` and share it to both Web and Build. We should avoid having this duplicated.